web/src/main/java/org/springframework/security/web/csrf/CookieCsrfTokenRepository.java

```diff
@@ -63,7 +65,18 @@ public final class CookieCsrfTokenRepository implements CsrfTokenRepository {
 
 	private int cookieMaxAge = -1;
 
-	public CookieCsrfTokenRepository() {
+	@Nullable
+	private Consumer<ResponseCookie.ResponseCookieBuilder> cookieInitializer = null;
```

Comment:
Spring Security tends to follow the null object pattern instead of making things nullable. Are you able to change this to be set `(builder) -> {}`?

Reply:
Sure. I made the change.👍🏻